R/createBinAnnotations.R

```diff
@@ -173,14 +173,7 @@ calculateBlacklist <- function(bins, bedFiles, ncpus=1) {
                 max(start, overlaps[i, "start"]) + 1
         bases / (end - start + 1) * 100
     }
-    if (ncpus > 1L) {
-        snowfall::sfInit(parallel=TRUE, cpus=ncpus)
-        snowfall::sfExport(list=c("overlap.counter"))
-        blacklist <- snowfall::sfApply(bins, 1, overlap.counter, joined)
-        snowfall::sfStop()
-    } else {
-        blacklist <- apply(bins, MARGIN=1L, FUN=overlap.counter, joined)
-    }
+    blacklist <- fapply(bins, MARGIN=1L, FUN=overlap.counter, joined)
```

Comment:
This makes the `ncpus` parameter obsolete. Would the best practice be to set the default to `ncpus=NULL`, and then in the code do something like `if (!is.null(ncpus)) warning("Parameter ncpus is deprecated.")`. And then remove it altogether for the next Bioconductor version?